config/google_drive_config.json

```diff
@@ -5,5 +5,5 @@
     "https://www.googleapis.com/auth/drive",
     "https://spreadsheets.google.com/feeds/"
   ],
-  "refresh_token": "1/AUNh1aOdTESQBa4f4xqE5qVHNGnO-_2dEi4hF1woPYdIgOrJDtdun6zK6XiATCKT"
-}
+  "refresh_token": "1/NMmpVCcuqR5f1kaHxqKKCOqNJeeqzvzjhIvKW8ep-0I"
+}
```

Comment:
Sorry, I should have made myself clear - I was referring to the lack of terminating newline.